dnf-behave-tests/fixtures/specs/comps-environment-upgrade-1/comps.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE comps PUBLIC "-//Red Hat, Inc.//DTD Comps info//EN" "comps.dtd">
+<comps>
+
+  <group>
+   <id>testgroup</id>
+   <default>false</default>
+   <uservisible>true</uservisible>
+   <display_order>1024</display_order>
+   <name>testgroup</name>
+   <description>Testgroup for DNF CI testing</description>
+    <packagelist>
+      <packagereq type="mandatory">mandatory</packagereq>
+      <packagereq type="default">default</packagereq>
+      <packagereq type="optional">optional</packagereq>
+      <packagereq type="conditional" requires="dummy">conditional-true</packagereq>
+      <packagereq type="conditional" requires="nonexistent">conditional-false</packagereq>
+    </packagelist>
+  </group>
```

Comment:
Is the group definition and the actual packages the same as in the group tests? Perhaps it would actually be better to share the two repositories for the tests?

you could likely even put them into the same feature file to maintain the naming convention of feature file and repo name...

On more of a side note, I think it would be a good idea to put the comps tests into their own directory under `features`, eventually...

Reply:
> Is the group definition and the actual packages the same as in the group tests? Perhaps it would actually be better to share the two repositories for the tests?

Not sure what you mean. The comps-group.feature uses dnf-ci-thirdparty and dnf-ci-fedora and it's more messy. But I can change it so that both features use the same group-specific repositories.

> you could likely even put them into the same feature file to maintain the naming convention of feature file and repo name...
> 
> On more of a side note, I think it would be a good idea to put the comps tests into their own directory under `features`, eventually...

Makes sense.

Reply:
> > Is the group definition and the actual packages the same as in the group tests? Perhaps it would actually be better to share the two repositories for the tests?
> 
> Not sure what you mean. The comps-group.feature uses dnf-ci-thirdparty and dnf-ci-fedora and it's more messy. But I can change it so that both features use the same group-specific repositories.

I mean the `testgroup` comps group definition as well as the RPMs in these two commits appear to be identical (though I didn't compare them side by side): https://github.com/rpm-software-management/ci-dnf-stack/pull/885/commits/4b7d83b492fd3ed9d8912659477380faf913fdee, https://github.com/rpm-software-management/ci-dnf-stack/pull/885/commits/0ce0601d8c57b63190be1208ef2a864c5a596a4f

The only difference then being the presence of the env group, and so I think the whole two repositories could be shared between the comps group upgrade and comps env upgrade tests?

Reply:
Oh, I see. Another difference is having different group in the environment in comps-environment-upgrade-2 to tests that new group will be installed. But yeah, it could be simplified.

Reply:
I unified the repositories and added both group and environment tests into one feature.